src/components/ui/Checkbox.module.scss

```
@@ -0,0 +1,49 @@
+.checkbox {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 13px;
+  height: 13px;
+
+  border-radius: 6px;
+  background-color: rgba(255, 255, 255, 0.1);
+  transition: 0.2s;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "transition" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)